robots:
  c_bot:
    default_gripper: c_bot_gripper
  d_bot:
    default_gripper: d_bot_gripper

grippers:
  c_bot_gripper:
    type: RobotiqGripper
    args:
      prefix: c_bot_gripper_
      effort: 5.0
      velocity: 0.1
      timeout: 6.0
  d_bot_gripper:
    type: RobotiqGripper
    args:
      prefix: d_bot_gripper_
      effort: 5.0
      velocity: 0.1
      timeout: 6.0
